Add files via upload

ddo00 opened the pull request.

ddo00 committed:
Add files via upload

menuet commented on src/bulls_and_cows_lib/game.cpp:
```diff
@@ -1,91 +1,141 @@
+            cout << "\nType your guess: "; // TODO securiser la saisie (Entre A Et H, 5 charactere obligatoire)
+            cin >> test;
+            cout << "\nYour guess is: " << test;
+
```
Il vaudrait mieux mettre les code qui calcule les Bulls et les Cows dans une fonction et appeler cette fonction ici

menuet commented on src/bulls_and_cows_lib/game.cpp:
```diff
@@ -1,91 +1,141 @@
+        int i_bulls = 0;
+        int i_cows = 0;
+
+        while (i_bulls != 5)
```
Je suppose que tu vas enrichir le test de sortie de la boucle avec le cas où le joueur dépasse le nombre de tentatives maximales autorisées

menuet commented on src/bulls_and_cows_lib/game.cpp:
```diff
@@ -1,91 +1,141 @@
+                    i_bulls++;
+                }
+            }
+            // compteur de cows
```
As-tu pu tester le code qui compte les Cows, n'as-tu pas remarqué un bug ?

menuet commented on src/bulls_and_cows_lib/game.cpp:
```diff
@@ -1,91 +1,141 @@
+#include <fstream>
+#include <iostream>
+#include <thread>
+using namespace std;
```
Il vaut mieux éviter autant que possible d'utiliser **using namespace**, surtout pour le namespace std qui n'est que 3 lettre à taper.

menuet reviewed (commented):
Bon début, mais il reste beaucoup à faire, courage :-)

ddo00 committed:
Antoine R. option 1

L'option 3 n'est pas finie.

l'avancement de l'option 3 n'est pas présent dans ce commit pour une meilleur clarté du code 

J'ai tenu compte de vos remarques:
-j'ai éviter autant que possible d'utiliser using namespace, surtout pour le namespace std.
-j'ai effectivement remarquer un bug pour le compteur des Cows, il est corrigé
-le test de sortie de la boucle prend en compte le cas où le joueur dépasse le nombre de tentatives autorisées
-Le code qui calcule les Bulls et les Cows sont maintenant dans une fonction
